### PR TITLE
Navbar update

### LINKED ISSUE #17903: Odd Looking NavBar Button color when the screen size is reduced (hamburger)
# Improve documentation

## Link 
https://supabase.com/

Add a link to the page which needs improvement (if relevant)

## Describe the problem
The hamburger icons color is looking odd
Is the documentation missing? Or is it confusing? Why is it confusing?

## Describe the improvement
The color of it should be green or white
supabasecolor2

A clear and concise description of the improvement.

## Additional context

Add any other context or screenshots that help clarify your question.

### PR DESCRIPTION
**Pull request by fsansalvadore**
## What kind of change does this PR introduce?

- New navbar dropdowns
- Improved navbar a11y
- Improved mobile navigation
- GitHub star count
- Fixes #17903
- Addresses #13777

## What is the current behavior?

Screenshot 2023-09-28 at 18 25 43
Screenshot 2023-09-28 at 17 53 47
Screenshot 2023-09-28 at 17 55 10

### CODE REVIEW

File: apps/www/data/CustomerStories.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+export const data = [
+  {
+    type: 'Customer Story',
```
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply apps/www/data/CustomerStories.ts:3]: Yeah that was also my thought, but we'd need to create a HOC and use it on all pages which could add some maintenance burden. I'll see if I can abstract it enough.

File: packages/ui/src/components/shadcn/ui/navigation-menu.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ const NavigationMenuViewport = React.forwardRef<
   <div className={cn('absolute left-0 top-full flex justify-center')}>
     <NavigationMenuPrimitive.Viewport
       className={cn(
-        'origin-top-center relative mt-1.5 h-[var(--radix-navigation-menu-viewport-height)] w-full overflow-hidden rounded-md border bg-overlay text-popover-foreground shadow-lg data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-90 md:w-[var(--radix-navigation-menu-viewport-width)]',
+        'menu-viewport origin-top-center relative mt-1.5 h-[var(--radix-navigation-menu-viewport-height)] w-full overflow-hidden rounded-md border bg-overlay text-popover-foreground shadow-lg data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-90 md:w-[var(--radix-navigation-menu-viewport-width)]',
```
[the remark this answers is not in the document]
Comment (fsansalvadore) [reply packages/ui/src/components/shadcn/ui/navigation-menu.tsx:83]: I think that NavigationMenu needs to be extended a bit for `NavigationMenuViewport` to actually be accessible from the api.

**Comment by saltcod** [comment]
Love it @fsansalvadore 
Small thing — feels like there's a delay on the hover (maybe b/c there is?) 
The community dropdown here https://linear.app, while not a megamenu like this, feels like the right speed to me

**Comment by fsansalvadore** [comment]
> Small thing — feels like there's a delay on the hover (maybe b/c there is?)
The community dropdown here https://linear.app/, while not a megamenu like this, feels like the right speed to me

@saltcod I assume you meant the "Company" menu item from linear.

Yeah, the radix component we're using has a default delay of 200ms. I think it's to avoid opening the menu if you're just randomly passing by with the curso, making it open only when intentionally hovering on it, which makes sense.

With that said, I changed it to 0ms now. 😁
I agree with you that it feels snappier. 👌

File: apps/www/data/CustomerStories.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/data/CustomerStories.ts:3]: ah yea, maybe best left for a next13 improvement, could use a RSC to handle it.

**Review by MildTomato — APPROVED** [review]

**Review by MildTomato — APPROVED** [review]

**Pull request merged**